velox/exec/tests/TableWriteTest.cpp

```diff
@@ -93,6 +94,31 @@ class TableWriteTest : public HiveConnectorTestBase {
     return getRecursiveFiles(directoryPath).size();
   }
 
+  // Returns a table insert plan node.
+  // Helper method used for "immutablePartitions" set of tests.
+  PlanNodePtr createInsertPlan(
+      const RowTypePtr& rowType,
+      const std::vector<RowVectorPtr>& input,
```

Comment:
Please move the input parameter first in the arguments list since all the rest are related to each other.

Reply:
Updated.